build/common.sh

```diff
@@ -242,6 +241,7 @@ fi
 
 # load device-specific environment
 . ${DEVICEDIR}/${PRODUCT_DEVICE_REAL}.conf
+export SRCABI="FreeBSD:${SRCREVISION%%.*}:${PRODUCT_ARCH}"
```

Comment:
I would move the whole block handing SRC* including the comment... that way it looks like it was meant to be ;)

Reply:
Agreed, makes sense.